integration_tests/profiles/bigquery/profiles.yml

```diff
@@ -1,13 +0,0 @@
-default:
```

Comment:
Why are the profiles being removed? Aren't they needed for CI testing?

Reply:
there is a separate ci testing folder that has those profiles, these werent being used